packages/docs/render-cards.mjs

```diff
@@ -2,19 +2,130 @@ import { bundle } from "@remotion/bundler";
 import { getCompositions, renderStill } from "@remotion/renderer";
 import path from "path";
 
-const serveUrl = await bundle({
+// render cards
+const serveUrl_1 = await bundle({
   entryPoint: path.join(process.cwd(), "./src/remotion/entry.ts"),
   publicDir: path.join(process.cwd(), "static"),
 });
-const compositions = await getCompositions(serveUrl);
+const compositions_1 = await getCompositions(serveUrl_1);
 
-for (const composition of compositions.filter(
+for (const composition of compositions_1.filter(
   (c) => c.id.startsWith("expert") || c.id.startsWith("template")
 )) {
   await renderStill({
     composition,
     output: `static/generated/${composition.id}.png`,
-    serveUrl,
+    serveUrl: serveUrl_1,
   });
   console.log("Rendered", composition.id);
 }
+
+
+// extract articles
+import { execSync } from "child_process";
+import fs from "fs";
+import { readDir } from "./get-pages.mjs";
+
+const root = path.join(process.cwd(), "docs");
+
+const pages = readDir(root);
+
+const findId = (split, page) => {
+  const found = split.find((s) => s.startsWith("id: "));
+  if (found) {
+    return found.substr("id: ".length);
+  }
+
+  return page
+    .replace(process.cwd() + path.sep + "docs" + path.sep, "")
+    .replace(/.md$/, "");
+};
+
+const findTitle = (split) => {
+  const title = split
+    .find((s) => s.startsWith("title: "))
+    .replace(/^title:\s/, "");
+  if (title.startsWith('"')) {
+    return title.substr(1, title.length - 2);
+  }
+
+  return title;
+};
+
+const findCrumb = (split) => {
+  const crumb = split
+    .find((s) => s.startsWith("crumb: "))
+    ?.replace(/^crumb:\s/, "");
+  if (crumb?.startsWith('"')) {
+    return crumb.substr(1, crumb.length - 2);
+  }
+
+  return crumb ?? null;
+};
+
+const data = [];
+
+for (const page of pages) {
+  const opened = fs.readFileSync(page, "utf8");
+  const frontmatter = opened.match(/---\n((.|\n)*?)---\n/);
+  if (!frontmatter) {
+    continue;
+  }
+
+  const split = frontmatter[1].split("\n");
+  const id = findId(split, page);
+  const title = findTitle(split);
+  const crumb = findCrumb(split);
+
+  const relativePath = page.replace(process.cwd() + path.sep, "");
+  const compId =
+    "articles-" + relativePath.replace(/\//g, "-").replace(/.md$/, "");
+  data.push({ id, title, relativePath, compId, crumb });
+}
+
+fs.writeFileSync(
+  path.join(process.cwd(), "src", "data", "articles.ts"),
+  `export const articles = ` + JSON.stringify(data, null, 2)
+);
+
+execSync("pnpm exec prettier src/data/articles.ts --write");
+
+const serveUrl_2 = await bundle({
+  entryPoint: path.join(process.cwd(), "./src/remotion/entry.ts"),
+  publicDir: path.join(process.cwd(), "static"),
+});
+const compositions_2 = await getCompositions(serveUrl_2);
```

Comment:
There is no need to call `bundle()` and `getCompositions()` twice - they do the same thing and can be reused actually!